diff --git a/crates/initramfs/dracut/bootc-root-setup.sh b/crates/initramfs/dracut/bootc-root-setup.sh
new file mode 100644
index 00000000..495f5772
--- /dev/null
+++ b/crates/initramfs/dracut/bootc-root-setup.sh
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+command -v getarg > /dev/null || . /lib/dracut-lib.sh
+
+mount_root() {
+    /usr/lib/bootc/initramfs-setup "x$NEWROOT"
+}
+
+mount_root
diff --git a/crates/initramfs/dracut/module-setup.sh b/crates/initramfs/dracut/module-setup.sh
index 39dfe238..458a97c4 100755
--- a/crates/initramfs/dracut/module-setup.sh
+++ b/crates/initramfs/dracut/module-setup.sh
@@ -9,10 +9,6 @@ depends() {
     return 0
 }
 install() {
-    local service=bootc-root-setup.service
     dracut_install /usr/lib/bootc/initramfs-setup
-    inst_simple "${systemdsystemunitdir}/${service}"
-    mkdir -p "${initdir}${systemdsystemconfdir}/initrd-root-fs.target.wants"
-    ln_r "${systemdsystemunitdir}/${service}" \
-        "${systemdsystemconfdir}/initrd-root-fs.target.wants/${service}"
+    inst_hook pre-pivot 10 "$moddir/bootc-root-setup.sh"
 }
